### ISSUE TITLE
coc.nvim vimtex not loaded on markdown files

### ISSUE THREAD

**Issue by gennaro-tedesco** [neoclide/coc-vimtex#30]
## Setup
```
NVIM v0.6.1
Build type: Release
LuaJIT 2.1.0-beta3
```
on 
```
macOS 10.16.0 Big Sur
```

## Problem
Interestingly enough this extension outputs an exception when opening `.md` files, in particular
```
[coc.nvim] vimtex not loaded
```
As per [examples](https://github.com/neoclide/coc-vimtex#options) I have explicitly specified 
```
...
coc.source.vimtex.filetypes: ["tex", "latex"],
...
```
only, in my CoC options, nevertheless the exception persists. Shall I include further options? I see a [similar ticket](https://github.com/neoclide/coc-vimtex/issues/28#event-5814189257) but it is unclear to me what the solution is: a see the fix commit - has it not been released yet?

**Comment by scarlehoff** [comment]
Indeed, I don't think it is released yet.
But if you apply that commit to your coc-vimtex installation (for me is in `.config/coc/extensions/node_modules`, I don't know where is it in Mac) the markdown files won't try to enable vimtex anymore.

**Comment by chemzqm** [comment]
Published coc-vimtex@1.1.1
